ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.h

```diff
@@ -11,6 +11,25 @@
 #import <Foundation/Foundation.h>
 
 struct CKComponentLayout;
+@class CKComponent;
+
+/** Represents a component scope collision. contains the component with the collision, the lowest common ancestor of the colliding components and the backtrace description */
```

Comment:
I'd add a brief one sentence summary here and add docblocks on each of the three members. As-is this line is a bit long.